src/libraries/System.Security.Cryptography.Xml/tests/SignatureSupport.cs

```diff
@@ -0,0 +1,20 @@
+namespace System.Security.Cryptography.Xml.Tests
+{
+    class SignatureSupport
+    {
+        private static int _supportsRsaSha1Signatures = 0;
+
+        public static bool SupportsRsaSha1Signatures
+        {
+            get
+            {
+                if (_supportsRsaSha1Signatures == 0)
+                {
+                    bool supported = System.Security.Cryptography.Tests.SignatureSupport.CanProduceSha1Signature(new RSACryptoServiceProvider());
```

Comment:
Is it possible to imagine a product bug that broke the functionality would cause this to fail, causing the tests you annotated to not run on any platform, concealing the bug?

Reply:
This library should just include that same file from Common.